etl_manager/meta.py

```diff
@@ -42,28 +42,59 @@ class TableMeta :
         'datetime' : {'glue' : 'double', 'spark': 'DoubleType'},
         'boolean' : {'glue' : 'boolean', 'spark': 'BooleanType'}
     }
-
-    def __init__(self, filepath, database = None) :
-        meta = _read_json(filepath)
-        self.columns = meta['columns']
-        self.name = meta['table_name']
-        self.description = meta['table_desc']
-        self.data_format = meta['data_format']
-        self.id = meta['id']
-        self.location = meta['location']
-
-        if 'partitions' in meta :
-            self.partitions = meta['partitions']
+    def __init__(self, filepath, **kwargs) :
```

Comment:
I would include a default arg:  `filepath=None`.  I think it makes sense that if you don't include a filepath you get a blank slate

Reply:
Sure